content/blog/2020-12-30-december-20-community-gems.md

```diff
@@ -0,0 +1,206 @@
+---
+title: December '20 Community Gems
+date: 2020-12-30
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, read about custom DVC plots, teamwork with DVC, CML without Docker, and maintaining several pipelines in parallel!
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, read about custom DVC plots, teamwork with DVC, CML without Docker, and maintaining several pipelines in parallel!
+picture: 2020-12-30/cover.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/november-20-community-gems/566
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Cache
+  - Bitbucket
+---
+
+## DVC questions
+
+### [Q: Is there a way to plot all columns in a `.csv` file on a single graph using `dvc plot`?](https://discord.com/channels/485586884165107732/563406153334128681/768689062314770442)
+
+By default, `dvc plot` graphs one or two columns from the metric file of your
+choice (use the `-x` and `-y` flags to specify which columns).
+
+However, there's nothing special about the way DVC makes plots. The plot
+function is a wrapper for the [Vega-Lite](https://vega.github.io/vega-lite-v1/)
+grammar, which can make pretty much any kind of plot you can imagine. If you
+check inside `.dvc/plots/`, you'll see a few Vega-Lite template files- that's
+where the plotting instructions are stored!
+
+You can create your own, or modify the existing templates, by
```

Comment:
we can mention some details from @pared 's answer as well - https://discord.com/channels/485586884165107732/563406153334128681/768733088535674931